2020-08-28 v. 1.0.8: added distribute flavors

vipozdn opened the pull request:
1. Added:
- app/src/main/... MainActivity class;
- app/src/thirdpartyStores/... MainActivity class.

2. Updated:
buildd.gradle (app) file
- added flavors,
- Play Core Library depenency changed to only googlePlay-flavor implementation.

3. Deleted:
- unused Play Core Library imports in app/src/thirdpartyStores/... MainActivity class.

## Checklist

__Common__

- [x] I am ran the app before creating PR
- [x] I am ran all tests before creating PR

__UI__

- [x] I am ran the app for visual checks.

__Logic__

- [x] I am tested basic app functionality.

## Changes

Describe all changes here:

- First;
- Second;
- Third;
- ...

## Comments

Describe all additional information here.

vipozdn committed:
2020-08-28 v. 1.0.8: added distribute flavors

1. Added:
- app/src/main/... MainActivity class;
- app/src/thirdpartyStores/... MainActivity class.

2. Updated:
buildd.gradle (app) file
- added flavors,
- Play Core Library depenency changed to only googlePlay-flavor implementation.

3. Deleted:
- unused Play Core Library imports in app/src/thirdpartyStores/... MainActivity class.

fartem commented on app/src/googlePlay/java/com/smlnskgmail/jaman/hashcheckerlite/MainActivity.java:
```diff
@@ -7,6 +7,7 @@
 import android.content.IntentSender;
 import android.net.Uri;
 import android.os.Bundle;
+import android.util.Log;
```
[Codacy](https://app.codacy.com/gh/hash-checker/hash-checker-lite/pullRequest?prid=7989232) find issue here. Please, fix it.

vipozdn replied:
done ✓

fartem reviewed (changes requested):
Check [Codacy](https://app.codacy.com/gh/hash-checker/hash-checker-lite/pullRequest?prid=7989232) issues.

vipozdn committed:
2020-09-01 v. 1.0.8: deleted unused import

1. Deleted:
- unused import statement in app/src/googlePlay.../MainActivity.java

fartem commented:
@vipozdn thanks.
